platform-sdk/swirlds-state-api/src/test/java/com/swirlds/state/spi/FilteredReadableStatesTest.java

```diff
@@ -47,21 +47,21 @@ void contains() {
         }
 
         @Test
-        @DisplayName("Throws IAE for any non-null key")
-        void nonNullKey() {
+        @DisplayName("Throws IAE for an unknown K/V state ID")
+        void unknownKVStateId() {
             assertThatThrownBy(() -> states.get(UNKNOWN_STATE_ID)).isInstanceOf(IllegalArgumentException.class);
         }
 
         @Test
-        @DisplayName("Throws IAE for any non-null Singleton key")
-        void nonNullSingletonKey() {
+        @DisplayName("Throws IAE for an unknown queue state ID")
```

Comment:
Display name is written for a queue, but should be for singleton

Reply:
Fixed